### PR TITLE
Fixed issue with attachments in Bulk

### PR DESCRIPTION
**Pull request by if0s**
https://github.com/elasticio/salesforce-component-v2/issues/57

### CODE REVIEW

File: lib/actions/bulk_cud.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 const { messages } = require('elasticio-node');
-const { Readable } = require('stream');
-const util = require('../util');
+// const { Readable } = require('stream');
```
Comment (A3a3e1) [inline lib/actions/bulk_cud.js:2]: commentaries

File: lib/actions/bulk_cud.js [hunk 2]
```diff
@@ -20,27 +21,26 @@ exports.objectTypes = async function objectTypes(configuration) {
...
-  const key = Object.keys(message.attachments)[0];
 
   let timeout = parseInt(configuration.timeout, 10);
   if (Number.isNaN(timeout) || (timeout < 0)) {
     this.logger.warn(`Timeout is incorrect. Set default value: ${DEFAULT_TIMEOUT} sec.`);
     timeout = DEFAULT_TIMEOUT;
   }
+  let { attachmentUrl } = message.body;
+  if (!attachmentUrl) {
+    if (!message.attachments || Object.keys(message.attachments).length === 0 || !message.attachments[Object.keys(message.attachments)[0]].url) {
```
Comment (A3a3e1) [inline lib/actions/bulk_cud.js:35]: Can't this be simplified to 
!message.attachments[0].url?

Comment (if0s) [reply lib/actions/bulk_cud.js:35]: `message.attachments` is an object where file name is a key and this object mast have `url` key inside

File: lib/actions/bulk_cud.js (hunk 1 above)
Comment (if0s) [reply lib/actions/bulk_cud.js]: removed

File: lib/actions/bulk_cud.js (hunk 2 above)
Comment (A3a3e1) [reply lib/actions/bulk_cud.js:34]: Ok. Got it. I was thinking of the wrong message.attachment structure. Then it's fine. But I'd better increase readability moving it into a method with a self-explanatory name.

**Review by A3a3e1 — APPROVED** [review]

**Pull request merged**